src/assignments/html-me-something.rst

```diff
@@ -3,14 +3,272 @@
+   your own content.
+#. Don’t add any CSS yet. Really, we mean it! If you think your page
+   looks boring without any style rules, that’s okay. You will fix that in Part
+   2.
+#. Use the HTML references linked on this page, or find others online. We
+   haven’t taught you every detail about every tag that you may want or need.
+#. You’re free to use tags that haven’t been introduced in class! We’ve given
+   you enough background to get started, but you are more than ready to do
+   some learning on your own.
+
+Halfway There!
+^^^^^^^^^^^^^^
+
+You finished Part 1! Copy the link to your repl.it or Trinket file and share it
+with your teacher.
+
+You will complete the second half of this project once you reach the end of the
+CSS chapter.
+
+.. todo:: Add internal link to the CSS chapter here.
+
+.. _html-me-part2:
 
 Part 2: CSS
 -----------
 
-Lorem ipsum...
+In Part 2, you will use CSS selectors and style rules to control what your page
+looks like. Remember, the goal is to keep your styling separate from your
+content.
+
+When you first created this project, you added a ``style.css`` file. Click on
+its name to open it in the editor. Right now, it should be empty. Most of your
+code for Part 2 will go inside ``style.css``.
+
+Now start adding some CSS rules!
+
+Requirements
+^^^^^^^^^^^^
+
+#. Be creative! Make your page look great, and don’t just settle for checking
+   off the items from this requirements list.
+#. Use at least one of each of the following selectors:
+
+   a. `element <http://www.w3schools.com/cssref/sel_element.asp>`__,
+   b. `class <http://www.w3schools.com/cssref/sel_class.asp>`__,
+   c. `id <http://www.w3schools.com/cssref/sel_id.asp>`__.
+
+   Remember, ``class`` and ``id`` selectors require you to add attributes
+   inside a starting tag.
+
+   .. todo:: Replace external links with internal LCHS chapter links (CSS selectors).
+
+#. Avoid adding HTML elements in order to achieve a specific visual effect. Do
+   ALL of your styling with CSS.
+#. Include `margin <http://www.w3schools.com/css/css_margin.asp>`__ and
+   `padding <http://www.w3schools.com/css/css_padding.asp>`__ in the
+   declaration blocks to space your elements in a visually pleasing way.
+#. Use inline ``style`` attributes only when absolutely necessary.
+
+Part 2 Tips
+^^^^^^^^^^^
+
+#. In order to see any visible change, make sure to link ``style.css`` to your
+   ``index.html`` file.
+#. Feel free to check out our `styled example
+   <http://education.launchcode.org/html-me-something/submissions/chrisbay/index.html>`__
+   to see how we did things. Right-click anywhere on the page and select
+   *View Page Source*. In the tab that opens, click on the ``style.css`` link
+   on line 8.
+#. Check out the `W3Schools CSS Reference <http://www.w3schools.com/css/default.asp>`__.
+
+Done!
+-----
+
+You completed assignment 4! Take a moment to gaze with pride at your webpage.
```

Comment:
consider removing both of the exclamations here too. They are really getting a lot of use in this assignment - I think if you want to convey anticipation or excitement on this page, it can be a stronger effect to use additional language rather than the exclamation point

Reply:
Fixed, thanks!